MantidPlot/src/Mantid/MantidUI.cpp

```diff
@@ -3230,6 +3230,7 @@ MultiLayer* MantidUI::plot1D(const QMultiMap<QString,int>& toPlot, bool spectrum
     g->checkValuesInAxisRange(firstCurve);
   }
   ml->toggleWaterfall(waterfallPlot);
+  g->setAutoScale();
```

Comment:
Is this going to override whether autoscaling is enabled/disabled by the user preferences?

A few lines above (#L3220) there is a comment to that effect.